### PR TITLE
Update tutorial-archive.md

### PR DESCRIPTION
**Pull request by Yael-sys**
Besides correcting typos and grammar, I have also:
1. Added an introduction to KQL.
2. Added explanations about the operators, where needed.
3. Reordered some of the sections (for instance, moved the 'take' operator before the 'project' operator.

I would suggest adding a third level of Heading, and group sections which belong to the same operator there (for instance, the 'render' variations).

### CODE REVIEW

**Comment by Court72** [comment]
@Yael-sys 

Thank you for your contribution. Would you take a moment to sign the [Contributor License Agreement (CLA)](https://github.com/MicrosoftDocs/dataexplorer-docs/pull/2181#issuecomment-1950929524)? After the CLA is signed, someone can review your pull request. 

Thanks!

**Comment by Court72** [comment]
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/tutorial-archive.md [hunk 1]
```diff
@@ -8,62 +8,67 @@ ms.date: 11/01/2021
...
+|StartTime|EndTime|EpisodeId|EventId|State|EventType|...|
+|--|--|--|--|--|--|--|
+|2007-09-20T21:57:00Z|2007-09-20T22:05:00Z|11078|60913|FLORIDA|Tornado|...|
+|2007-12-20T07:50:00Z|2007-12-20T07:53:00Z|12554|68796|MISSISSIPPI|Thunderstorm Wind|...|
+|2007-12-30T16:00:00Z|2007-12-30T16:05:00Z|11749|64588|GEORGIA|Thunderstorm Wind|...|
+|2007-09-29T08:11:00Z|2007-09-29T08:11:00Z|11091|61032|ATLANTIC SOUTH|Waterspout|...|
+|2007-09-18T20:00:00Z|2007-09-19T18:00:00Z|11074|60904|FLORIDA|Heavy Rain|...|
 
-## Show *n* rows: *take*
+Note: [limit](./take-operator.md) is an alias for [take](./take-operator.md) and has the same effect.
```
Comment (Jak-MS) [inline data-explorer/kusto/query/tutorial-archive.md:57]:
```suggestion
> [!Note]
> [limit](./take-operator.md) is an alias for [take](./take-operator.md) and has the same effect.
```

**Comment by Jak-MS** [comment]
@shsagir 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

Note that the validation warnings must be addressed - the links fixed - before we can merge this.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Pull request closed without merging**